core/modules/replica/Configuration.h

```diff
@@ -270,6 +270,77 @@ class Configuration {
     /// @return the name of a database to be set upon the connection
     std::string const& databaseName() const { return _databaseName; }
 
+    /// @return the number of concurrent connections to the database service
+    size_t databaseServicesPoolSize() const { return _databaseServicesPoolSize; }
+
+    // --------------------------------------------------
+    // -- Global parameters of the database connectors --
+    // --------------------------------------------------
+
+    /**
+     * @return the default mode for database reconnects.
+      */
+    static bool databaseAllowReconnect() { return defaultDatabaseAllowReconnect; }
+
+    /**
+     * Change the default value of the parameter
```

Comment:
`Change the default value of the parameter` has a pretty low signal to noise ratio. I'd choose between a more helpful comment and removing it if the description is not needed.

Reply:
Perhaps :-)

Reply:
Fixed